storybook/src/components/form/form.stories.ts

```diff
@@ -44,12 +44,6 @@ storiesOfForm(
     formGroupTextareaTemplate,
   }),
   {
-    formGroupDecorator: (story) =>
-      html`<form>
-        <fieldset>
-          <legend>Example</legend>
-          ${story()}
-        </fieldset>
-      </form>`,
+    formGroupDecorator: (story) => html`<form>${story()}</form>`,
```

Comment:
Goed gevonden! Dit is de decorator voor de Form Groups. Een decorator is een concept van Storybook en daarmee kan je een story (markup) voorzien van extra markup.

Deze decorator wrapt de markup (vanaf nu) alleen met een `form` element.

Idealiter zouden we hier de `formTemplate()` gebruiken, maar zo werkt Storybook niet. Dus dit is goed, maar je moet nog in `form.css-template.ts` dezelfde wijziging toepassen.